source/content/apcu.md

```diff
@@ -0,0 +1,38 @@
+---
+title: APCu Cache Basics
+description: Understand APCu and its uses within the Pantheon WebOps workflow.
+categories: [platform]
+tags: [cache, workflow]
+reviewed: "2020-10-08"
+---
+
+The **APCu** is a user-accessible PHP cache. PHP is a dynamic language that is compiled on-demand into bytecode at execution time. To improve performance, APCu stores this bytecode so that it can be reused instead of having to be recompiled each time.
```

Comment:
> To improve performance, APCu stores this bytecode so that it can be reused instead of having to be recompiled each time.

This is another opcode cache reference that we should drop.